src/common/shared.ts

```diff
@@ -5,16 +5,36 @@ const { ipcRenderer: ipc } = window.require("electron-better-ipc");
 
 export const COMMAND_SELECTOR_OPTIONS = [
   {
-    value: "textContent",
-    text: "Text content",
+    value: "isHidden",
+    text: "Check Hidden",
   },
   {
     value: "isVisible",
     text: "Check Visibility",
   },
   {
-    value: "isHidden",
-    text: "Check Hidden",
+    value: "innerText",
```

Comment:
nit: keep innertext and textContent close to one another and other boolean checks before.

Reply:
Will do.